herald/base.py

```diff
@@ -224,8 +237,17 @@ def get_extra_data(self):
 
         return extra_data
 
+    def get_attachments(self):
+        """
+        Return a list of attachments or None.    
+
+        This only works with email.
+        """
+        return self.attachments
+
     @staticmethod
-    def _send(recipients, text_content=None, html_content=None, sent_from=None, subject=None, extra_data=None):
+    def _send(recipients, text_content=None, html_content=None, sent_from=None, subject=None, extra_data=None,
+              attachments=None):
```

Comment:
@georgmzimmer you added the attachments arg here, but you didn't add it to the BaseNotification class or the TwilioNotification class. They should all have the same method signature